db_stress_tool/db_stress_driver.cc

```diff
@@ -174,10 +180,18 @@ bool RunStressTestImpl(SharedState* shared) {
     }
   }
 
-  for (unsigned int i = 1; i < n; i++) {
-    threads[0]->stats.Merge(threads[i]->stats);
+  // If we are running post_verification_only
```

Comment:
Should we also skip `stress->PrintStatistics();` in line 206?

Reply:
Thanks a lot @cbi42. I added a conditional around it.